yoeo/detect.py

```diff
@@ -27,7 +27,7 @@
 
 def detect_directory(model_path, weights_path, img_path, classes, output_path,
                      batch_size=8, img_size=416, n_cpu=8, conf_thres=0.5, nms_thres=0.5,
-                     multi_robot=False, first_robot_id=0):
+                     robot_class_ids: Optional[List[int]] = None):
```

Comment:
Optional and list are not imported

Reply:
https://github.com/bit-bots/YOEO/blob/eb6bd857df945465641662cf7edd07dbe1763623/yoeo/detect.py#L15 ?

Reply:
Weird, I am pretty sure GitHub showed me an linter error yesterday and the import line was missing. Maybe it showed only a subset of the commits idk..